src/plugins/rovpilot-wire/index.js

```diff
@@ -44,7 +44,7 @@
     deps.globalEventLoop.on('mcu.status', function (status) {
       if ('targetDepth' in status) {
         self.state.depth.enabled = status.targetDepth != DISABLED;
-        self.state.depth.targetDepth = Number(status.targetDepth) / 100;
+        self.state.depth.targetDepth = Number(status.targetDepth);
```

Comment:
Ah. Charles and I were wondering the reason for that. Why is the depth hold in centimeters and the depth in meters?